src/test/java/com/exasol/parquetio/reader/RowParquetChunkReaderTest.java

```diff
@@ -1,55 +1,67 @@
 package com.exasol.parquetio.reader;
 
+import static org.hamcrest.CoreMatchers.containsString;
+import static org.hamcrest.CoreMatchers.equalTo;
+import static org.hamcrest.CoreMatchers.startsWith;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.containsInAnyOrder;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.concurrent.atomic.AtomicInteger;
+
 import com.exasol.parquetio.data.ChunkInterval;
 import com.exasol.parquetio.data.ChunkIntervalImpl;
 import com.exasol.parquetio.data.GenericRow;
 import com.exasol.parquetio.data.Row;
 import com.exasol.parquetio.writer.ParquetTestFileWriter;
+
 import org.apache.parquet.io.InputFile;
 import org.apache.parquet.io.ParquetDecodingException;
 import org.apache.parquet.io.RecordReader;
+import org.apache.parquet.io.SeekableInputStream;
 import org.apache.parquet.io.api.RecordMaterializer;
 import org.apache.parquet.schema.PrimitiveType;
 import org.junit.jupiter.api.Test;
 import org.junit.jupiter.api.io.TempDir;
-
-import java.io.IOException;
-import java.nio.file.Files;
-import java.nio.file.Path;
-import java.util.ArrayList;
-import java.util.Collections;
-import java.util.List;
-import java.util.concurrent.atomic.AtomicInteger;
-
-import static org.hamcrest.CoreMatchers.*;
-import static org.hamcrest.MatcherAssert.assertThat;
-import static org.hamcrest.Matchers.containsInAnyOrder;
-import static org.junit.jupiter.api.Assertions.assertThrows;
-import static org.mockito.Mockito.mock;
-import static org.mockito.Mockito.when;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.NullAndEmptySource;
 
 // [utest->dsn~read-parquet-file-chunks-contents~1]
 class RowParquetChunkReaderTest {
 
     private static final int RECORD_COUNT = 200000;
 
+    @ParameterizedTest
+    @NullAndEmptySource
+    void testReadChunksNullAndEmptyThrows(final List<ChunkInterval> list) {
+        final var inputFile = new SimpleInputFile("dummy");
+        final IllegalArgumentException exception = assertThrows(IllegalArgumentException.class,
+                () -> new RowParquetChunkReader(inputFile, list));
+        assertThat(exception.getMessage(), startsWith("E-PIOJ-5"));
```

Comment:
I also keep repeating these two lines and created a util class for this.
AssertJ has a convenient way for assertions like this. Would it be OK to add it as a dependency?

Example: 
```java
        assertThatThrownBy(() -> foo.doStuff())
                .isInstanceOf(FooException.class)
                .hasMessage("An exception has occurred")
                .hasCauseExactlyInstanceOf(IllegalStateException.class);
```